### PR TITLE
Performance Improvements

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

File: src/Tribe/Timezones.php [hunk 1]
```diff
@@ -29,6 +29,17 @@ public static function init() {
 	protected static function invalidate_caches() {
 		add_filter( 'pre_update_option_gmt_offset', array( __CLASS__, 'clear_site_timezone_abbr' ) );
 		add_filter( 'pre_update_option_timezone_string', array( __CLASS__, 'clear_site_timezone_abbr' ) );
+		add_action( 'update_option_gmt_offset', [ __CLASS__, 'clear_timezone_cache' ] );
```
Comment (tr1b0t) [inline src/Tribe/Timezones.php]: :no_entry_sign: Short array syntax is not allowed

<sub>hash: 9cfd502cfc3ac8032ed42b589e329e2e</sub>

File: src/Tribe/Timezones.php [hunk 2]
```diff
@@ -29,6 +29,17 @@ public static function init() {
...
+		add_action( 'update_option_timezone_string', [ __CLASS__, 'clear_timezone_cache' ] );
```
Comment (tr1b0t) [inline src/Tribe/Timezones.php]: :no_entry_sign: Short array syntax is not allowed

<sub>hash: 3ce9a92baba6ce23e4e2dea2783acb5e</sub>

File: src/Tribe/Timezones.php [hunk 3]
```diff
@@ -66,22 +77,29 @@ public static function wp_timezone_abbr( $date ) {
 	 * @return string
 	 */
 	public static function wp_timezone_string() {
+		static $cache_key = __METHOD__;
+		$cached_value = tribe_get_var( $cache_key );
```
Comment (tr1b0t) [inline src/Tribe/Timezones.php]: :warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: 0285facfa9fc32bee2d5b851b477e316</sub>

**Comment by borkweb** [comment]
Unless these are bugfixes for our performance changes, this should be pull requested against the 5.0.1 release branch

**Comment by bordoni** [comment]
Yeah for sure, they are meant for 5.0.1, my mistake.

File: src/Tribe/Container.php [hunk 4]
```diff
@@ -185,12 +197,17 @@ function tribe( $slug_or_class = null ) {
...
+	 *
 	 * @param string $slug  The human-readable and catchy name of the var.
 	 * @param mixed  $value The variable value.
+	 *
+	 * @return mixed       Returns the set value, which allows for shorter code.
 	 */
 	function tribe_set_var( $slug, $value ) {
 		$container = Tribe__Container::init();
 		$container->setVar( $slug, $value );
+		return $value;
```
Comment (sc0ttkclark) [inline src/Tribe/Container.php]: Useful!

**Pull request closed without merging**